controllers/http.go

```diff
@@ -176,6 +199,9 @@ func registryScanCommandToScanCommand(c wssc.RegistryScanCommand) domain.ScanCom
 		Args:            c.Args,
 		Session:         sessionChainToSession(c.Session),
 	}
+	if slug, err := names.ImageInfoToSlug(c.ImageTag, "nohash"); err == nil {
+		command.ImageID = slug
+	}
```

Comment:
question: is it okay to pass this `nohash` value down to the SBOM creator?

Doesn’t this pollute the data? Will Syft be okay scanning images with the `nohash` tag?

Reply:
it's fine